OWML.ModLoader/ModLoader.cs

```diff
@@ -36,6 +37,18 @@ public static void LoadMods()
             owo.LoadMods();
         }
 
+        private static IModConsole GetConsole(IOwmlConfig owmlConfig, IModLogger logger, IModManifest owmlManifest)
+        {
+            if (CommandLineArguments.HasArgument(Constants.ConsolePortArgument))
+            {
+                return new ModSocketConsole(logger, owmlManifest);
+            }
+            else
+            {
+                return new ModConsole(owmlConfig, logger, owmlManifest);
+            }
+        }
```

Comment:
This exact method also exists in Owo.cs . Where should I move it to?

Reply:
Well... it's very tempting to introduce a new project for everything related to logging/output at this point. It could be called OWML.ModHelper.Logger or similar, and contain ModConsole, ModSocketConsole, and this helper method. Both the console app and OWML uses ModHelper.
However, I'm not sure if it makes sense to ask you to include that in your PR. Do you want to?

Reply:
...and ModLogger.